--- drbd-0.7.22.orig/user/Makefile	2004-10-28 12:44:25.000000000 +0000
+++ drbd-0.7.22.Makefile/user/Makefile	2006-11-23 23:22:42.000000000 +0000
@@ -17,7 +17,7 @@
 # the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
 #
 
-CFLAGS = -c -Wall -I../drbd
+CFLAGS = $(OPTCFLAGS) -Wall -I../drbd
 CC = gcc
 
 drbdadm-obj = drbdadm_scanner.o drbdadm_parser.o drbdadm_main.o \
@@ -28,7 +28,7 @@
 all: drbdsetup drbdadm
 
 drbdadm: $(drbdadm-obj)
-	$(CC) -o $@ $^ 
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
 
 drbdadm_scanner.c: drbdadm_scanner.fl drbdadm_parser.h
 	flex -s -odrbdadm_scanner.c drbdadm_scanner.fl
@@ -42,7 +42,7 @@
 	bison -d -o drbdadm_parser.c drbdadm_parser.y
 
 drbdsetup: $(drbdsetup-obj)
-	$(CC) -o $@ $^ 
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^
 
 clean:
 	rm -f drbdadm_scanner.c
